Library/Services/Implementation/NfieldLocalUserService.cs

```diff
@@ -82,6 +82,17 @@ public async Task<LocalUser> UpdateAsync(string identityId, LocalUserValues mode
             }
         }
 
+        public async Task ResetAsync(string identityId, ChangePasswordLocalUser model)
+        {
+            if (model == null)
+                throw new ArgumentNullException(nameof(model));
+
+            var uri = new Uri(ConnectionClient.NfieldServerUri, $"LocalUsers/Password/{identityId}");
+            using (await ConnectionClient.Client.PatchAsJsonAsync(uri, model).ConfigureAwait(false))
+            {
+            }
```

Comment:
Why an empty using?

Reply:
I think for dispose it. It's done everywhere in the api